examples/ex_C4.jl

```diff
@@ -12,86 +12,38 @@ let m = SOSModel(SCS.Optimizer)
     @variable m t
     @objective m Max t
     @variable m sos SOSPoly([1; x])
-    @constraint m sum(x) + sum(x.^2) - t == sos
+    @constraint m sum(x) + sum(x .^ 2) - t == sos
```

Comment:
`^` is an exception to the rule that there should be space around binary operators:
https://jump.dev/JuMP.jl/v0.21.6/style/#Exceptions

Reply:
as long as this is not part of JuMP, I'd rather rely on automatic formatting for those details.